definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -0,0 +1,142 @@
+[
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solAosReceivedNoAdConStarted",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solAosSubmittedUndefended",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solConfirmPersonalService",
+    "UserRole": "[PETSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorDraftAOS",
+    "UserRole": "[PETSOLICITOR]",
```

Comment:
this needs R for idam solicitor role as well.

Reply:
yep, here you can put "R" for just the idam role so it will be inherited automatically by [PETSOLICITOR]

Reply:
Just to clarify, I believe that AC 15/16/17 and 18 all require a minimum R for the idam case role - "caseworker-divorce-solicitor"
For `solicitorDraftAOS`, `solicitorUpdateAOS`, `solicitorSubmitAOS`, `solicitorApplyForDN` none of them have the IDAM role set to R.
@yarpo As this goes against the ACs, can I get conformation that this is correct? And if so, could you update the story accordingly?